web/index.css

```
@@ -2,248 +2,1412 @@
+
+/* ═══════════════════════════════════════════════════════════════════════════
+   B · Brutalist Terminal (default)
+   Pure black. Geist Mono. #00ff66. 1px gaps. Left-border select.
+   ═══════════════════════════════════════════════════════════════════════════ */
+
+:root,
+[data-theme="brutalist"] {
+  color-scheme: dark;
+
+  /* Font stack */
+  --gs-font: 'Geist Mono', 'JetBrains Mono', 'SF Mono', Monaco, 'Cascadia Code', monospace;
```

Stylelint (17.5.0)

[error] Expected "Monaco" to be "monaco" (value-keyword-case)

(value-keyword-case)